docs/deployment/azure/customize-deployments.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Customize .NET Aspire deployments with infrastructure synthesis
+description: Learn how to customize Azure deployments using azd infra synth for production scenarios.
+ms.date: 01/08/2025
+ms.custom: devx-track-extended-azdevcli
+---
+
+# Customize .NET Aspire deployments with infrastructure synthesis
+
+The Azure Developer CLI (`azd`) provides a powerful feature called infrastructure synthesis that allows you to generate and customize the underlying infrastructure code for your .NET Aspire applications. This capability is essential for production scenarios where you need fine-grained control over Azure resources, security configurations, and deployment patterns.
+
+This article covers how to use `azd infra synth` to:
```

Comment:
You'll need to update all places where you reference `azd infra synth` as it's been renamed to `azd infra gen` (short for `generate`) - see: https://github.com/Azure/azure-dev/pull/5260.

```suggestion
This article covers how to use `azd infra gen` to:
```